doc/sphinx/proofs/writing-proofs/rewriting.rst

```diff
@@ -90,11 +91,24 @@ Rewriting with Leibniz and setoid equality
      any of them can be rewritten.  If not specified, only the first occurrence
      in the conclusion is replaced.
 
-     If :n:`at @occs_nums` is specified, rewriting is always done with
-     :ref:`setoid rewriting <generalizedrewriting>`, even for Leibniz’s equality,
-     which means that you must `Require Setoid` to use that form.
-     However, note that :tacn:`rewrite` (even when using setoid rewriting) and
-     :tacn:`setoid_rewrite` don't behave identically (as already mentioned above).
+     .. note::
+
+        If :n:`at @occs_nums` is specified, rewriting is always done
+        with :ref:`setoid rewriting <generalizedrewriting>`, even for
+        :term:`Leibniz equality`, which means that you must `Require
+        Setoid` to use that form.  However, note that :tacn:`rewrite`
+        (even when using setoid rewriting) and :tacn:`setoid_rewrite`
+        don't behave identically (as already mentioned above).
+
+     .. note::
+
+        For each selected hypothesis (and the conclusion),
+        :tacn:`rewrite` will instantiate the rewriting lemma first
+        before taking the `at` clause into account, which limits the
+        number of occurrences that can be rewritten (see `#13780
+        <https://github.com/coq/coq/issues/13780>`_).  You can force
+        :tacn:`rewrite` to instantiate the rewriting lemma differently
+        by using a `with` clause (see :n:`@one_term_with_bindings`).
```

Comment:
An example would be better than pointing to an issue. By the way, there are other duplicates of this issue, e.g.,  #2564.

Reply:
Updated with an example and more information extracted from #2564.
Thanks!